rules/windows/image_load/image_load_phantom_load_oci_hijacking_via_msdtc_service.yml

```diff
@@ -0,0 +1,27 @@
+title: Phantom DLL Hijacking via MSDTC Service
+id: a3a8042a-5a91-46c0-ba14-c36c77ff529c
+status: experimental
+description: |
+    Detects phantom DLL hijacking technique where the non-existent DLL 'oci.dll' is loaded by the 'msdtc.exe' service.
+    Threat actors may place a malicious 'oci.dll' in a location where 'msdtc.exe' looks for it, leading to execution of the malicious DLL with elevated privileges of the MSDTC service.
```

Comment:
I mean this is already covered by a3a8042a-5a91-46c0-ba14-c36c77ff529c and we dont need it.